config/routes.php

```diff
@@ -20,6 +20,7 @@
 
 use Cake\Core\Plugin;
 use Cake\Routing\RouteBuilder;
+use Cake\Routing\Route\DashedRouted;
```

Comment:
Also phpcs complains that this class should be listed last as per alphabetical order :P